frontend/src/app/modules/auth/auth-page/auth-page.component.sass

```diff
@@ -50,85 +55,93 @@
         text-shadow: -2px 2px 1px $pink-100
     .car-background
         @include background-icon
-        right: 30%
-        top: 46px
+        right: 28%
+        top: 6%
     .car
+        @include icon-size
         position: absolute
-        right: 31.3%
-        top: 50px
+        right: 27%
+        top: 7%
     .camera-background
         @include background-icon
-        left: 1217px
-        top: 101px
+        right: 7%
+        top: 18%
     .camera
+        @include icon-size
         position: absolute
-        left: 1235px
-        top: 118px
+        right: 6%
+        top: 20%
     .clover-background
         @include background-icon
-        left: 1267px
-        top: 315px
+        right: 4%
+        top: 48%
     .clover
+        @include icon-size
         position: absolute
-        left: 1282px
-        top: 333px
+        right: 3%
+        top: 50%
     .cook-background
         @include background-icon
-        left: 1113px
-        top: 441px
+        right: 18%
+        top: 68%
     .cook
+        @include icon-size
         position: absolute
-        left: 1128px
-        top: 455px
+        right: 17%
+        top: 70%
     .dancing-background
         @include background-icon
-        left: 1003px
-        top: 551px
+        right: 26%
+        top: 82%
         opacity: 0.2
     .dancing
+        @include icon-size
         position: absolute
-        left: 1020px
-        top: 569px
+        right: 25%
+        top: 84%
     .dog-background
         @include background-icon
-        right: 39%
-        top: 451px
+        right: 40%
+        top: 62%
     .dog
+        @include icon-size
         position: absolute
```

Comment:
For the future - it's better not to use position:absolute and position:fixed in your code, unless you need to make some difficult components, which is not the case. For now, let's leave it